docs/querying/sql-functions.md

```diff
@@ -1039,11 +1057,35 @@ Returns a string representing an approximation to the histogram given the specif
 
 ## DS_HLL
 
-`DS_HLL(expr, [lgK, tgtHllType])`
+Creates an HLL sketch on a column containing HLL sketches or a regular column. The [HLL sketch documentation](../development/extensions-core/datasketches-hll.md) describes the optional `lgK` and `tgtHllType` arguments.
```

Comment:
Same comment as https://github.com/edgar2020/druid/pull/4/files#r1753076652. Also, this should say parameters instead of arguments.